### DISCUSSION TITLE
Could I make a proxy server with Xrdp ?

### DISCUSSION THREAD

**Discussion by seunggyu0572** [neutrinolabs/xrdp#2312] [Q&A]
I wanted to make a proxy server.
but client is microsoft remote desktop and server is window.

Could I make a proxy server with Xrdp ?

**Comment by TOMATO-ONE** [comment]
Hello.

You can do it!
I have been using xrdp as RDP Proxy for several years now.

Build [NeutrinoRDP](https://github.com/neutrinolabs/NeutrinoRDP), install the libraries.
Then run xrdp with `./configure --enable-neutrinordp` to build it.
The RDP Proxy module `libxrdpneutrinordp.so` is generated.

This will enable the `[neutrinordp-any]` section of `xrdp.ini` to function and be used as an RDP Proxy.
